faker/seed_data.py

```diff
@@ -93,21 +111,31 @@ def add_patient_assignment_to_bed(database_connection: sqlite3.Connection) -> No
 
             all_patient_ids.remove(random_patient)
 
+        logging.info(f"Assigned some patients to beds in db")
+
     database_connection.commit()
-    print("Data generation info: Assigned some patients to beds in db")
 
 
 if __name__ == "__main__":
     random.seed(43)
     if not check_data_existence("../db/hospital.db"):
+        clear_database("../db/hospital.db")
+
+        logging.basicConfig(
+            stream=sys.stdout,
+            level=logging.INFO,
+            format="%(asctime)s - %(levelname)s - %(message)s",
+            force=True,
+        )
+
```

Comment:
It's fine right now but in the future, this can be improved

https://youtu.be/9L77QExPmI0?si=M_ja06B9JhR_tycC.